apps/docs/src/common/pages/accessibility/guidance.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <Helmet>
+      <title>Accessibility - Home Office Design System</title>
+      <meta name="og:article:section" content="Accessibility" />
+    </Helmet>
+    <div className="govuk-grid-column-one-quarter">
+      <NavigationMenu items={[
+        {
+          href: '/accessibility/audio-and-video',
+          text: 'Audio and video'
+        },
+        {
+          href: '/accessibility/guidance',
+          text: 'Guidance'
+        },
+        {
+          href: '/accessibility/tools',
+          text: 'Tools'
+        },
+        {
+          href: '/accessibility/further-reading',
+          text: 'Further reading'
+        }
+      ]} />
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+    <h1>
+      <span className="caption">Accessibility</span>
+      Guidance
+    </h1>
+    <ul className="plain">
+      <li><A href="https://collaboration.homeoffice.gov.uk/display/A11y/Accessibility+Standard">Home Office Accessibility Standard v1.0 Nov 2020</A></li>
```

Comment:
@elliot-butler can we put in brackets after the first link: (Home Office Confluence document)
Second link: (Home Office Sharepoint document)